expertise/service/expertise.py

```diff
@@ -315,8 +288,9 @@ def get_expertise_results(self, user_id, job_id, delete_on_get=False):
 
         # Clear directory
         if delete_on_get:
-            self.logger.info(f'Deleting {search_dir}')
-            shutil.rmtree(search_dir)
+            self.logger.info(f'Deleting {config.job_dir}')
+            shutil.rmtree(config.job_dir)
+            JobConfig.remove_job(user_id, job_id, self.redis_args)
```

Comment:
Rather than explicitly removing the job config from Redis, you can just set an expiry date for each config key.

Reply:
Never mind, It's for `delete_on_get`, but either way, I think we should add an expiration date for config, something like a week or two? What do you think @melisabok?

Reply:
after the job is finished?

Reply:
I think this is a good idea, but I am concerned about the mismatch between configs in Redis and its related files on the disk. Maybe Redis has a handler that is run when an object expires that also lets us clear the job data?

Reply:
Right, Redis has a property called `ttl` to specify the expiration date. 

But I'm not sure if I want to keep the job data in the files, aren't we going to clean the jobs folder regularly?